### PR TITLE
Updates for the Developer Portal

### PR DESCRIPTION
**Pull request by navinger**
Updates so the sample API can be used easily in the Developer Portal. 
-- In schemes, it has to be "http" so that you can test the echo API in the Developer Portal. 
-- In the echoMessage definition, adding type: "object" provides a better user experience in the Developer Portal.

### CODE REVIEW

File: endpoints/getting-started/openapi.yaml [hunk 1]
```diff
@@ -25,7 +25,9 @@ consumes:
 produces:
   - "application/json"
 schemes:
-  - "https"
+# Uncomment the next line if you configure SSL for this API.
```
Comment (lesv) [inline endpoints/getting-started/openapi.yaml]: And comment out the `http` line

**Comment by navinger** [comment]
I asked an engineer on the developer portal team if you really do have to comment out the http line, and his response was:
"no, multiple schemes are OK
it's not either/or"

Please merge this change to master. You can't test the sample code in the Developer Portal without this change.

**Comment by lesv** [comment]
SGTM - though typically users want to lock down http when they use https.  We are encouraging folks to go http only.

**Comment by lesv** [comment]
@dpebot merge when green

**Pull request merged**